core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackService.java

```diff
@@ -142,6 +142,11 @@
     public static final int NOTIFICATION_TYPE_PLAYBACK_SPEED_CHANGE = 8;
 
     /**
+     * Playback speed has changed
```

Comment:
both this and the comment above say 'playback speed has changed'.  

I assume this new comment is wrong.

The variable name is awkward, but I can't think of a better one, so a better comment would be good.